internal/variants/variants.go

```diff
@@ -89,9 +120,19 @@ func (q *Query) whereClause() string {
 	}
 	simpleClause("reference_name", q.RefName)
 	simpleClause("reference_bases", q.Allele)
-	// Start is inclusive, End is exclusive.  Search exactly for coordinate.
-	if q.Coord != nil {
-		add("v.start <= %d AND %d < v.end", *q.Coord, *q.Coord+1)
-	}
+	q.bqCoordinatesToWhereClause(add)
```

Comment:
Aren't start/end just simpleClauses?

And then you could probably inline the last line of the bqCoordinates... function